phantomgames/Servicelayer/ConstantEndpoints.swift

```diff
@@ -0,0 +1,12 @@
+//
```

Comment:
why are there two constants file?
This can just be added to the other constants file :)

Reply:
why was this resolved with no feed back?

Reply:
my apologies, I added the the endpoints to the other other constant file and now only have 1 constant file